src/Agent.Worker/TaskCommandExtension.cs

```diff
@@ -232,6 +236,33 @@ private void ProcessTaskUploadSummaryCommand(IExecutionContext context, string d
             }
         }
 
+        private void ProcessTaskUploadFileCommand(IExecutionContext context, string data)
```

Comment:
@chrisrpatterson, do you want to give this a glance over w.r.t. how this command is implemented? Seems fine as far as I can tell.